source/modules/MEI.xml

```diff
@@ -2019,6 +2019,9 @@
         <valItem ident="cut">
           <desc>Cut time; i.e. 2/2.</desc>
         </valItem>
+        <valItem ident="senza">
+          <desc>Senza misura</desc>
```

Comment:
How does this differ from the `invis` value in `data.METERFORM`?

Reply:
Please see #731. It is meant to specifically encode [this symbol](https://smufl-browser.edirom.de/index.html?glyphname=timeSigX):
 
![](https://smufl-browser.edirom.de/E09C.png)

See Gould pp. 611–615

Reply:
Ah, Ok. I was going by this:

https://usermanuals.musicxml.com/MusicXML/Content/EL-MusicXML-senza-misura.htm

Given that the other options in this are the "cut" and "common", perhaps something like "open" (from the SMuFL description "Open time signature") or just "x"? (not that I'm necessarily a fan of single-letter attribute values, but it is the most descriptive)  

Given that the MusicXML interpretation is an invisible time signature, but you definitely intend a visible one, maybe making that more explicit is a good thing. We can still keep "senza misura" in the description to make it clear...

Reply:
Also this example, where the time signature has been hidden:

https://repertoire-explorer.musikmph.de/en/product/holst-gustav-17/

Reply:
I agree. `open` is a better value here.

Reply:
Only in music notation does "X" mean "open", but it is what it is. :) 

Thanks!